### PR TITLE
Clarify orphaned volume cleanup

### PR DESCRIPTION
**Pull request by pmorie**
Clarify log messages for and make consistent the cleanup of orphaned volumes

@kubernetes/sig-node

### CODE REVIEW

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -1987,31 +1987,32 @@ func (kl *Kubelet) cleanupOrphanedVolumes(pods []*api.Pod, runningPods []*kubeco
...
+			glog.V(3).Infof("Orphaned volume %q found, tearing down volume", name)
 			// TODO(yifan): Refactor this hacky string manipulation.
 			kl.volumeManager.DeleteVolumes(types.UID(parts[0]))
 			// Get path reference count
-			refs, err := mount.GetMountRefs(kl.mounter, cleanerTuple.Unmounter.GetPath())
+			volumePath := cleanerTuple.Unmounter.GetPath()
+			refs, err := mount.GetMountRefs(kl.mounter, volumePath)
 			if err != nil {
-				return fmt.Errorf("Could not get mount path references %v", err)
+				glog.Errorf("Could not get mount path references for %q: %v", volumePath, err)
```
Comment (yujuhong) [inline pkg/kubelet/kubelet.go]: The existing code returns here? Did you change this intentionally?

Comment (pmorie) [inline pkg/kubelet/kubelet.go]: I did change it intentionally, and was going to add a note that it seems arbitrary (and undesirable) that we should return here.  Returning an error here means we won't clean up other orphaned dirs if there's a problem with one.  We could also just accumulate errors and return a single error that wraps them if there's a problem at the end of the list.

TLDR: I think we should attempt to clean up every directory here instead of short circuiting for this one particular error condition.

Comment (yujuhong) [inline pkg/kubelet/kubelet.go]: This sounds reasonable. I was just asking because the rest of PR involves only logging changes :)
We should make all other cleanup functions to have the same behavior.

**Comment by yujuhong** [comment]
LGTM. Please rebase and thanks for the PR!
